pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.black]
+target-version = ['py38', 'py39', 'py310']
+skip-string-normalization = false
+line-length = 79
+exclude = '''
+(
+  /(
+      \.eggs
+    | \.git
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | build
+    | dist
+    | examples
+  )/
+)
+'''
+
+[tool.isort]
```

Comment:
The only bit we really need here is `profile` https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg and the few things like `known_first_party`

Reply:
Does that ensure that, for example, editor integrations will call black if `isort` is not installed? (pre-commit will do it fine, but that also needs the developer to have followed the indications).